scheduler/kubernetes/etcd-certs-using-secrets.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: page
+title: "Etcd certificates using Kubernetes Secrets"
+keywords: portworx, container, kubernetes, storage, k8s, etcd, secrets, certificates
+meta-description: "This guide is a step-by-step tutorial on how to use Kubernetes secrets to give etcd certificates to Portworx."
+---
+
+This page will guide you on how to give your etcd certificates to Portworx using Kubernetes Secrets. This is the recommended way of providing etcd certificates, as the certificates will be automatically available to the new nodes joining the cluster.
+
+#### Create Kubernetes secret
+Copy all your etcd certificates and key in a directory `etcd-secrets/` to create a Kubernetes secret from it.
+```
+# ls -1 etcd-secrets/
+ca.crt
+client.crt
+client.key
+```
+
+Use `kubectl` to create the secret named `px-etcd-certs` from the above files:
+```
+# kubectl -n kube-system create secret generic px-etcd-certs --from-file=etcd-secrets/
+```
+
+Notice that the secret has 3 keys `ca.crt`, `client.crt` and `client.key`, that coincide with their corresponding file names. We will use these keys in the Portworx spec file to reference the certificates.
```

Comment:
Let's not use the word `coincide` since this isn't a coincidence but rather something that will always happen.